site/en/docs/extensions/reference/readingList/index.md

```diff
@@ -0,0 +1,101 @@
+---
+api: readingList
+has_warning: This permission <a href="/docs/extensions/mv3/permission_warnings/#permissions_with_warnings">triggers a warning</a>.
+---
+
+Chrome features a reading list which allows users to save web pages to read later or when offline.
+Use the Reading List API to retrieve existing items and to add and remove items from the list.
+
+<figure>
+  {% Img src="image/wVNVUJS8Z8O04i1tJKSdsp6nkRQ2/isx7uJZA6B8YmEGTFmB6.png", alt="Reading list showing a number of articles", width="400", height="371" %}
+  <figcaption>
+    Reading list showing a number of articles
+  </figcaption>
+</figure>
+
+## Manifest {: #manifest }
+
+To use the Reading List API, add the `"readingList"` permission in the extension [manifest][doc-manifest] file:
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  "name": "My reading list extension",
+  ...
+  "permissions": [
+    "readingList"
+  ]
+}
+```
+
+## Concepts and usage {: #concepts }
```

Comment:
Here's an idea, since the concepts are short and few, what do you think about moving them into the use cases as an aside or within the content. 

For example:
"Items in the reading list are not in any guarenteed order." could go as a note under **display items**
 
And "Items are keyed by URL. This includes the hash and query string." Could go under **Add an item**.

Thoughts?

Reply:
I think there is value to keeping them here actually. I feel like they are useful things you should know before doing anything with the API, vs things you only learn if you look at the "Add an item" section for example.

Reply:
I understand your logic, so let's go with that. Visually, I think it looks pretty empty, maybe a definition list would help. I'm going to approve since it's not a deal-breaker